### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Farida-Hamid**
> In this project, I added the Views associated with the Models and Controllers I've already created for the Blog App.

### Steps:
- Implement the design from the sneak peek wireframes.
- Add data to the database.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,11 @@
...
\ No newline at end of file
+<div>
+  <% @users.each do |user| %>
+    <div class="user_container">
+      <div class="photo"></div>
+      <div class="user-info">
+        <h1><%= user.name%></h1>
+        <h4>Number of posts: <%= user.postscounter%></h4>
+      </div>
+    </div>
```
Comment (Omar-Muhamad) [inline app/views/users/index.html.erb:9]: - [ ] Please make the name of the user a link when clicked it redirects you to the user show page so any user can browse your app.
[Hint] You can use [link_to](https://www.rubyguides.com/2019/05/rails-link_to-method/) method to do this.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+  <% @user.recent_posts.each_with_index do |post, index|%>
+    <div  class="post">
+    <h3>Post #<%= index+1%></h3>
+      <p><%= post.text%></p>
+      < class="like-comment">Comments: <%= post.comments_counter%> Likes: <%= post.likes_counter%></p>
+    </div>
+  <%end%>
+
+  <button>See all post</button>
```
Comment (Omar-Muhamad) [inline app/views/users/show.html.erb:22]: - [ ] Same here, Clicking the `see all posts` button suppose to redirect to user post index page, So please fix that.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,24 @@
...
+      <h4>Number of posts: <%= @user.postscounter%></h4>
+    </div>
+  </div>
+  <div>
+    <% @posts.each_with_index do |post, index|%>
+      <div class="post">
+        <h3>Post #<%= index+1%></h3>
+        <p><%= post.text%></p>
+      <p class="like-comment">Comments: <%= post.comments_counter%> Likes: <%= post.likes_counter%></p>
+      </div>
```
Comment (Omar-Muhamad) [inline app/views/posts/index.html.erb:15]: - [ ] Same here, Clicking on the post title suppose to redirect to the user post show page, So please fix that to help users browse your app.

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Farida-Hamid,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by danmainah — APPROVED** [review]
Hi @Farida-Hamid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

- [ ] [optional] @Farida-Hamid It would have been nice if you used the latest [microverse_template](https://github.com/microverseinc/readme-template) to document your project.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**